tools/amd_build/pyHIPIFY/hipify-python.py

```diff
@@ -712,6 +729,9 @@ def preprocessor(filepath, stats, hipify_caffe2):
         # Replace std:: with non-std:: versions
         output_source = replace_math_functions(output_source)
```

Comment:
@Jorghi12 We should remove this temp function since your transpilation logic is trying to do the same thing more smartly by only replacing in the device code.

Reply:
It's not 100% perfect yet, so planning to keep the old logic for now.